server/controllers/request.js

```diff
@@ -14,22 +14,25 @@ const getRequests = async (req, res) => {
 // @route POST
 
 const createRequest = async (req, res) => {
-  req.body.userId = req.user.id;
-  const request = await Request.create(req.body);
+  const { sitterId, start, end } = req.body;

```

Comment:
A good security practice on this one :+1: It will also be good to validate the request body so you can return a `400` on a missing field.